OSDOCS-16945#CQA work Stor1 - understanding persistent storage (#114120)
openshift/openshift-docs: modules/storage-persistent-storage-overview.adoc

= Persistent storage overview

[role="_abstract"]

> Persistent volumes (PVs) and persistent volume claims (PVCs) separate storage provisioning from consumption, enabling administrators to manage storage resources while developers request capacity and access modes independently.

Reviewer: ```suggestion
Persistent volumes (PVs) and persistent volume claims (PVCs) separate storage provisioning from consumption, so that administrators can manage storage resources while developers request capacity and access modes independently.
```
Small edit to avoid anthropomorphism.

ifndef::microshift[]
Managing storage is a distinct problem from managing compute resources. {product-title} uses the Kubernetes persistent volume (PV) framework to allow cluster administrators to provision persistent storage for a cluster. Developers can use persistent volume claims (PVCs) to request PV resources without having specific knowledge of the underlying storage infrastructure.

> PVCs are specific to a project, and are created and used by developers as a means to use a PV. PV resources on their own are not scoped to any single project; they can be shared across the entire {product-title} node and claimed from any project. After a PV is bound to a PVC, that PV cannot then be bound to additional PVCs. This has the effect of scoping a bound PV to a single namespace, that of the binding project.

Reviewer: Does the change from "cluster" to "node" need QE review? This isn't my area of expertise, so I'm not sure.

Author: @tedaveryredhat PTAL. I'm not sure why I changed this. Also, line 20 has is weird too: "PV resources on their own are not scoped to any single namespace; they can be shared across the entire {product-title} cluster..." Shouldn't this be "node" rather than "cluster" for MS?

Reviewer 2: @lpettyjo Yes, it should be "node" rather than "cluster" for MS.

Reviewer 2: @lpettyjo I've reviewed the existing persistent storage sections for OCP and MS on docs.redhat.com. Line 16 that begins with "PVCs are specific to a project,..." is specific to OCP and "{product-title} cluster" is used. In this instance, "cluster" is correct. Line 20 that begins with "PVCs are specific to a namespace,..." is specific to MS and  "{product-title} node" is used. In other words, I think "node" needs to be "cluster" in line 16 and "cluster" needs to be changed to "node" in line 20.
